client/src/components/layout/Alert.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+const Alert = ({ alerts }) => {
+  const alertTypeToStyle = t => {
+    switch (t) {
+      case 'success':
+        return 'alert-success';
+      case 'error':
+        return 'alert-error';
+      case 'warning':
+        return 'alert-warning';
+      default:
+        return 'alert-info';
+    }
```

Comment:
You may be wondering: why not just dynamically create the class string down below when declaring the styles for the alert `div`? Tailwind's purge utility removes all styles not being used in the code base when building for prod to drastically reduce the css size, and it does so by matching a regular expression (this is intentionally simple to catch cases like this...). So, our `alert-*` classes will get purged out if they don't appear in their full form statically.

Reply:
Great find! Although; perhaps that means there's a better way to declare classnames? Regardless, I think this is fine as is:)

Reply:
I'm open to suggestions :shrug: either way, there are four classes to switch between here so I figured it was best to keep it close to the point of use